bounties/npm/react-oauth-flow/1/README.md

```diff
@@ -0,0 +1,5 @@
+# Overview
+react-oauth-flow is a small library to simplify the use of OAuth2 authentication inside your react applications.
+This package has been deprecated by the maintainer.
```

Comment:
> This package has been deprecated by the maintainer

If the package is deprecated, I don't think we should merge this. Also, there is no mention of this project being deprecated on the README.